cmd/cbnt-prov/cmd.go

```diff
@@ -416,11 +416,13 @@ func (g *generateBPMCmd) Run(ctx *context) error {
 			se.DigestList.List[iterator].HashAlg = g.IbbHash[iterator]
 		}
 
-		seg := *bootpolicy.NewIBBSegment()
-		seg.Base = g.IbbSegbase
-		seg.Size = g.IbbSegsize
-		seg.Flags = g.IbbSegFlag
-		se.IBBSegments = append(se.IBBSegments, seg)
+		if g.IbbSegbase != 0 {
```

Comment:
I think this will break if you don't supply -iibsegbase=0

Reply:
I had the same issue several times in the past. If you check on the existence of an optional argument which wasn't supplied, it is not set at all and will result in a null pointer exception.

Reply:
This seems to work fine, as coreboot does not specify all arguments. kong does have a flags to set the default value of a command line argument. Should that be used?

Reply:
If it works it works

Reply:
Are variables not 0 initialized in go? That's probably why it works then for integers.